test/regression/aarch64/Exception.cc

```diff
@@ -7,6 +7,62 @@ namespace {
 
 using Exception = AArch64RegressionTest;
 
+/** AArch64 opcodes. Each opcode represents a unique AArch64 operation. */
+namespace Opcode {
+#define GET_INSTRINFO_ENUM
+#include "AArch64GenInstrInfo.inc"
+}  // namespace Opcode
+
+// Test that an invalid capstone instruction id raises an illegal instruction
+// exception
+TEST_P(Exception, illegal_instruction) {
```

Comment:
Not sure what this comment is trying to change. The `InstructionException::EncodingUnallocated` exception is used when an encoding is not supported by capstone (because it's invalid or too "new") which we then communicate to the user with the string "illegal instruction".

Are you trying to say that the "illegal instruction" string should be changed to "unallocated instruction encoding"? I see no issue with this change if this is what you're suggesting.

Reply:
I wasn't necessarily asking for a change, just wondering why there is the mismatch between the names. I think illegal instruction has a different meaning (it does in RISC-V anyway) so printing "unallocated instruction encoding" may be more precise and informative for a user so I would be happy to make that change

Reply:
I'll follow the reasoning so will make the change